modules/cos-coredns/versions.tf

```diff
@@ -20,11 +20,11 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 3.53, < 6"
```

Comment:
Why would we block to version 6 ?
We don't know what's included in version 6 yet, and the module may still work.
Currently, it's "< 5.0" , and it's blocking terraform setup to update to provider v5, it's the only terraform google module that I use that does this.